src/api/controllers/misc.ts

```diff
@@ -30,24 +30,18 @@ export async function handleInternalReady(
   cache.shardedGuilds += payload.guilds.length;
   eventHandlers.shardReady?.(shardID);
 
-  //Ability to fine tune
-  const chunk = 1200;
-  const buffer = 400;
-  const request = 7;
-
-  //Remaining guilds to load via handleInternalGuildCreate
-  function checkRemaining(left: any) {
-    //If higher than chunk, delete chunk value, otherwise set to 0
-    cache.shardedGuilds >= chunk
-      ? cache.shardedGuilds - chunk
+  const values: number[] = [1000, 9, 500];
```

Comment:
```suggestion
  const values: [number, number, number] = [1000, 9, 500];
```
A tuple is better-suited.